packages/terra-date-picker/CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 # Changelog
 
 ## Unreleased
-
+* Fixed
```

Comment:
Add a line break for consistency
```suggestion

* Fixed
```

Reply:
Done https://github.com/cerner/terra-framework/commit/acd9d0dfd0f8928b39ceb4dedbb003c77806849d